src/styles/components/MasterHeader.scss

```diff
@@ -53,6 +54,16 @@ nav.navbar{
   }
   .btn-link{
     height: 40px;
+    padding-left: 0;
+    padding-right: 0;
+    width: 100%;
+    text-align: left;
+
+    .caret {
+      position: absolute;
+      right: 12px;
+      top: 19px;
```

Comment:
im not entirely sure how i feel about position absolute and putting fixed pixels for it. I made this huge mistake in the previous version and only pained me alot. Especially when looking at mobile, etc. I'd hazard away from this please.